src/baseInterceptor.js

```diff
@@ -182,7 +182,7 @@ function RGBAString(arguments){
         }
       }
       return(getRGBAname(values));
-    }else{
+    }else if (((arguments[0].match(/%/g)).length)==3){
       //when arguments[0] is 'rgba(10%,100%,30%,0.5)'
```

Comment:
Can we remove the last argument?  //when arguments[0] is 'rgba(10%,100%,30%,0.5)'